articles/logs/references/log-event-type-codes.md

```diff
@@ -49,6 +49,7 @@ The following table lists the codes associated with the each log event.
 | `fepft` | Failed Exchange | Failed exchange of Password for Access Token | |
 | `fercft` | Failed Exchange | Failed Exchange of Password and MFA Recovery code for Access Token | |
 | `fertft` | Failed Exchange | Failed Exchange of <dfn data-key="refresh-token">Refresh Token</dfn> for Access Token | |
+| `febrtft` | Failed Exchange | Failed Exchange of Rotating <dfn data-key="refresh-token">Refresh Token</dfn> for Access Token | |
```

Comment:
Should this code match this one? https://github.com/auth0/manage/pull/4739/files#diff-842acf6928cd660b67fc6469ffa75d68R66

Reply:
Yep, fixed :-)